packages/pelagos/src/listInput/ListEntries.js

```diff
@@ -97,14 +131,20 @@ ListEntries.propTypes = {
 	highlightKey: PropTypes.string,
 	/** The data for the list. */
 	list: PropTypes.array,
-	/** Whether items are listed as columns. */
+	/** Whether items are reorderable. */
+	reorderable: PropTypes.bool,
+	/** Whether items are listed as columns. Defaults to true if reorderable */
```

Comment:
- Maybe "Set to true..." as I think "Defaults..." may imply that the developer can override it (which is not the case).
- The dot at the end is missing.